pkg/scheduler/testing/wrappers.go

```diff
@@ -619,6 +630,28 @@ func (p *PodWrapper) Req(resMap map[v1.ResourceName]string) *PodWrapper {
 	return p
```

Comment:
Rename this to `Res`, which means add both requests and limits to containers. Add another method `Req` for only requests.